doc/sphinxman/source/build_planning.rst

```diff
@@ -1144,16 +1168,39 @@ H. Build with explicit non-MKL LAPACK
 
   * Perhaps the best choice, if available, is Intel's MKL library,
     which includes efficient threaded BLAS and LAPACK (as of |PSIfour|
-    v1.1, earliest known working version is MKL 2013). On Mac, the
-    native Accelerate libraries are also recommended.
-
-  * For open-source LAPACK distributions, OpenBLAS (formerly GotoBLAS)
-    is known to work, while ATLAS is known
-    (https://github.com/psi4/psi4/issues/391) to have stability issues
-    with the DFOCC module.
+    v1.1, earliest known working version is MKL 2013). MKL, which is
+    freely available through conda, is the only BLAS/LAPACK distribution
+    fully supported by |PSIfour|.
+
+  * On Mac, the native Accelerate libraries are very nice and would
+    be recommended but for the potential conflict between |PSIfour|
+    BLAS and NumPy BLAS.
+
+  * The open-source LAPACK distributions OpenBLAS (formerly GotoBLAS)
+    mostly works. Use it at your own risk and after testing your
+    particular distribution, including tests run multithreaded,
+    if you intend to run |PSIfour| so. Use at least 0.2.15, and
+    pay attention to how it was compiled - unthreaded seems safe,
+    openmp-threaded is mostly safe, default pthreaded is *not* safe. See
+    https://github.com/psi4/psi4/issues/1009 for recent analysis.
+
+  * Another open-source LAPACK distribution, ATLAS had
+    stability issues with the DFOCC module at last testing,
+    https://github.com/psi4/psi4/issues/391.
 
   * ACML libraries are known to work with |PSIfour| v1.1 at ACML 6.
 
+* Because of how link loaders work, at runtime, the BLAS of |PSIfour|
+  and the BLAS of NumPy are not independent. There can be unpredictable
+  but reproducible numerical and thread-scaling errors if |PSIfour|
+  and NumPy BLAS don't match down to the library name (that is,
+  ``libmkl_rt``, ``libmkl_core.so``, ``libmkl_core.a`` are *not*
+  interchangeable). See https://github.com/psi4/psi4/issues/1007,
+  https://github.com/psi4/psi4/issues/748,
+  https://github.com/psi4/psi4/issues/755 for gory discussions.
+  Choose your NumPy and |PSIfour| compile conditions to use the same
+  BLAS distribution.
```

Comment:
This can be found in NumPy by the following:
```
>>> import numpy.distutils.system_info as sysinfo
>>> sysinfo.get_info('blas_opt')
{'libraries': ['mkl_rt', 'pthread'], 'library_dirs': ['/Users/daniel/anaconda3/lib'], 'define_macros': [('SCIPY_MKL_H', None), ('HAVE_CBLAS', None)], 'include_dirs': ['/usr/include', '/Users/daniel/anaconda3/include']}
```

Could possible have this as an "auto" in the future where BLAS is picked up from NumPy?

Reply:
do you happen to know if `include_dirs` will always have `mkl.h` if `libraries` is MKL?

Reply:
I dont know, seems like a question for the conda devs (who I think will say "yes"). Worth pip install NumPy to see what comes out as well.